src/Middleware/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs

```diff
@@ -60,7 +60,11 @@ public async Task RunClientTests()
 
         private static async Task<SamplesDeploymentResult> CreateDeployments(ILoggerFactory loggerFactory)
         {
-            var solutionPath = TestPathUtilities.GetSolutionRootDirectory("CORS");
+            var solutionPath = TestPathUtilities.GetSolutionRootDirectory("Middleware");
+
+            var runtimeFlavor = GetRuntimeFlavor();
```

Comment:
Hmm, these are the changes that are failing. It looks like the method doesn't exist. These two lines weren't part of my change.  Any idea why this is failing @natemcmaster? 
```
error CS0103: The name 'GetRuntimeFlavor' does not exist in the current context [D:\a\1\s\src\Middleware\CORS\test\FunctionalTests\FunctionalTests.csproj]
```

Reply:
Thanks for your help!